ci-operator/step-registry/ovn/conf/manifests/ovn-conf-manifests-commands.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
I meant -- *everyone* running the installer should ideally be using the same steps to do so, or we have drift. So this should underpin the IPI install flow, too.